src/FakeItEasy/ExceptionThrowerConfigurationExtensions.cs

```diff
@@ -130,6 +134,11 @@ public static IAfterCallSpecifiedConfiguration Throws<T1>(this IExceptionThrower
                 });
         }
 
+#if DISABLED
+
+        // This method already has a generic parameter, and it can't be inferred; if we add another, we make it unusable,
+        // especially since the new one is something unintelligible like IAnyCallConfigurationWithReturnTypeSpecificied<Foo>...
+

```

Comment:
Oh, and based on your original comment, is there a _need_ to hoist the `AssignsOutAndRefParameters…` methods up to the interfaces? I thought `Throws` was the only one that absolutely required it, and that the `Assigns…` methods would only cause difficulties if users specified the typesparams. I'm just making sure I understand. I'm not arguing for making extra difficulties for the users, just wondering if pushing the methods up to the interfaces is technically necessary in all cases.